packages/core/src/Page/constants.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   FOOTERS_FOLDER_PATH: '_markbind/footers',
   HEAD_FOLDER_PATH: '_markbind/head',
   HEADERS_FOLDER_PATH: '_markbind/headers',
+  LAYOUTS_FOLDER_PATH: '_markbind/layouts',
```

Comment:
there's one in `core/src/constants.js`

Reply:
is there a reason why these constants are separated?

Reply:
Its used in `Site.js` as well